packages/js/product-editor/changelog/update-do-not-allow-selecting-text-in-paginator

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Do not allow to select text in the Paginator component
```

Comment:
I think that this file should live under `packages/js/components/changelog`. This is why we see [this error here](https://github.com/woocommerce/woocommerce/actions/runs/6853041724/job/18632956482?pr=41405).

Reply:
Good! Updating cc @mattsherman